### ISSUE TITLE
VLC extension doesn't work well with file names with special characters

### ISSUE THREAD

**Issue by Maggie-123** [exebetche/vlsub#182]
I have a file that ends with 'Folie à deux'. I found the subtitle, downloaded and then nothing.
When I checked, the name had a strange character instead of 'à' so subtitle didn't match the file name and was not loaded.

**Comment by exebetche** [comment]
Yes, lua can't handle non ascii (english) character on windows in the video file or the folder path and vlc won't allow using external lua library to fix this problem. So you can either rename your file and parents folders, if necessary, ('Folie à deux' become 'Folie a deux') or use the "manual download" parameter in config to save the file manually.
